### PR TITLE
UBER-853 Add ydoc indexation

### PR DESCRIPTION
**Pull request by aonnikov**
# Contribution checklist

## Brief description
Required for https://github.com/hcengineering/uberflow/pull/158
Extendable attachment content indexation.

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: server/core/src/content.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import type { ContentAdapter, ContentAdapterFactory, ContentTextAdapter } from './types'
+
+import serverCore from './plugin'
+import { Readable } from 'stream'
+
+/**
+ * @public
+ */
+export async function createModelContentAdapter (
+  model: Tx[],
```
Comment (haiodo) [inline server/core/src/content.ts]: Why so difficult?

File: server/core/src/content.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  const adapters: Map<string, ContentAdapter> = new Map()
+
+  for (let tx of model) {
+    if (tx._class === core.class.TxCollectionCUD) {
+      tx = (tx as TxCollectionCUD<Doc, AttachedDoc>).tx
+    }
+    if (tx._class === core.class.TxCreateDoc) {
+      const createTx = tx as TxCreateDoc<Doc>
+      if (createTx.objectClass === serverCore.class.ContentAdapterFactory) {
+        const factory = (createTx as TxCreateDoc<ContentAdapterFactory>).attributes.factory
```
Comment (haiodo) [inline server/core/src/content.ts]: No, no, no, let's pass all of them in server pod configuration, we do not need to pass them with a model, they a part of server configuration not a model.

Comment (aonnikov) [reply server/core/src/content.ts]: Replaced with serve configuration

File: server/core/src/content.ts (hunk 1 above)
Comment (aonnikov) [reply server/core/src/content.ts]: Simplified

**Review by haiodo — APPROVED** [review]

**Pull request merged**